pom.xml

```diff
@@ -50,7 +50,7 @@
     <argLine>-Xmx768M -Djava.awt.headless=true</argLine>
     <jenkins.version>2.138.4</jenkins.version>
     <!-- Should only need to override these if using timestamped snapshots (but better to use Incrementals instead): -->
-    <jenkins-core.version>${jenkins.version}</jenkins-core.version>
```

Comment:
So this setting seems to be ignored. Wondering if this is because this is a indirection using a property, or for some other reason.
https://github.com/jenkinsci/plugin-pom/blob/956e0dea0c020ffcb77d88c15cf41b3dcb6ba602/.dependabot/config.yml#L14

Reply:
@dependabot can you explain/help us why this update is being proposed despite the ignore settings? Thanks!

Reply:
Yes - Dependabot is using the property name in its PR here, but the underlying dependency name that you need to ignore is `org.jenkins-ci.main:jenkins-core`. Sorry that's not clearer!

Reply:
Thanks! Trying this.